components/global/footer/Footer.tsx

```diff
@@ -1,127 +1,56 @@
-            <ul className="space-y-2">
-              {[
-                "Brug af bibliotekerne",
-                "Erstatninger og gebyrer",
-                "Opret bruger",
-                "Biblioteket - A til Å",
-                "Internetadgang",
-                "Print, scan og kopi",
-                "Booking af lokaler",
-                "Nyhedsbrev",
-              ].map(item => (
-                <li key={item}>
-                  <Link href="#" className="hover:underline">
-                    {item}
-                  </Link>
-                </li>
-              ))}
-            </ul>
-          </div>
-
-          <div>
-            <h2 className="mb-4 text-xl font-semibold">Online tilbud</h2>
-            <ul className="mb-8 space-y-2">
-              {["Litteratursiden", "Bibliotek.dk", "eReolen", "Filmstriben"].map(item => (
-                <li key={item}>
-                  <Link href="#" className="hover:underline">
-                    {item}
-                  </Link>
-                </li>
-              ))}
-            </ul>
-            <h2 className="mb-4 text-xl font-semibold">Text</h2>
-            <ul className="space-y-2">
-              {[
-                "Lorem ipsum a la lorem ip",
-                "Bibliotekerne lorem",
-                "Labore ipsum",
-                "Adipisci velit le lorem",
-              ].map(item => (
-                <li key={item}>
-                  <Link href="#" className="hover:underline">
-                    {item}
-                  </Link>
-                </li>
-              ))}
+        <div className="flex flex-col flex-nowrap lg:flex-row lg:justify-between">
+          <div className="col-span-full">
+            <h2 className="text-typo-subtitle-lg col-span-full mb-4">Information</h2>
+            <p className="max-w-[600px]">GO er børnenes digitale indgang til folkebiblioteket.</p>
+            <ul className="mt-10 grid gap-y-4">
+              <li>
+                <Link
+                  href="https://www.detdigitalefolkebibliotek.dk/ereolen-go"
+                  className="animate-text-underline">
+                  Info om GO
+                </Link>
+              </li>
+              <li>
+                <Link
+                  href="https://www.detdigitalefolkebibliotek.dk/ereolen-go-support"
+                  className="animate-text-underline">
+                  Kontakt og support
+                </Link>
+              </li>
+              <li>
+                <Link
+                  href="https://ereolengo.dk/inspiration/til-skoler"
+                  className="animate-text-underline">
+                  Skoler og UNI-login
+                </Link>
+              </li>
+              <li>
+                <Link
+                  href="https://ereolengo.dk/content/bliv-boganbefaler"
+                  className="animate-text-underline">
+                  Bliv boganbefaler
+                </Link>
```

Comment:
Please use the smart link component to ensure consistency in the application:

```suggestion
                <SmartLink
                  href="https://www.detdigitalefolkebibliotek.dk/ereolen-go"
                  className="animate-text-underline">
                  Info om GO
                </SmartLink>
              </li>
              <li>
                <SmartLink
                  href="https://www.detdigitalefolkebibliotek.dk/ereolen-go-support"
                  className="animate-text-underline">
                  Kontakt og support
                </SmartLink>
              </li>
              <li>
                <SmartLink
                  href="https://ereolengo.dk/inspiration/til-skoler"
                  className="animate-text-underline">
                  Skoler og UNI-login
                </SmartLink>
              </li>
              <li>
                <SmartLink
                  href="https://ereolengo.dk/content/bliv-boganbefaler"
                  className="animate-text-underline">
                  Bliv boganbefaler
                </SmartLink>
```